packages/react-aria-components/stories/Autocomplete.stories.tsx

```diff
@@ -235,7 +249,10 @@ export const AutocompleteAsyncLoadingExample = {
   render: (args) => {
     return <AsyncExample {...args} />;
   },
-  name: 'Autocomplete, useAsync level filtering'
+  name: 'Autocomplete, useAsync level filtering with load state',
+  args: {
+    includeLoadState: true
+  }
```

Comment:
When testing the async story, note this control. This will toggle the existance of a intermediate loading state as the list is filtered